run_tests.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -l
 
 
-currentLocation=$dirname $0
+currentLocation="$(cd "$(dirname "$0")"; pwd)"
```

Comment:
This isnt going to work in integration need to chase Andy Howell about whether this is sorted if not then this might need to be set back to what it was in order that it runs in integration.